roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: "Setting Docker Facts"
+  set_fact:
+    docker_storage_block_device: "{{ docker_storage_block_device | default(default_docker_storage_block_device) }}"
+    docker_storage_volume_group: "{{ docker_storage_volume_group | default(default_docker_storage_volume_group) }}"
+
+- name: "Install Docker"
```

Comment:
If this role is used exclusively for docker with OpenShift I would suggest just placing the docker-storage-setup file and then letting the OpenShift installer install docker and start the service. That way we potentially don't install a version that may be too new